next.config.js

```diff
@@ -1,11 +1,15 @@
-const withSass = require('@zeit/next-sass')
+const withSass = require("@zeit/next-sass");
 
-module.exports = withSass( {
+module.exports = withSass({
   webpack: config => {
     // Fixes npm packages that depend on `fs` module
     config.node = {
-      fs: 'empty'
-    }
-    return config
+      fs: "empty"
+    };
+    config.module.rules.push({
+      test: /\.svg$/,
+      use: ["@svgr/webpack"]
+    });
+    return config;
```

Comment:
This isn't necessary. Next.js handles loading images.

Reply:
That's right, sure!